packages/firebase_remote_config/lib/src/remote_config.dart

```diff
@@ -0,0 +1,210 @@
+part of firebase_remote_config;
+
+/// The entry point for accessing Remote Config.
+///
+/// You can get an instance by calling [RemoteConfig.instance]. Note
+/// [RemoteConfig.instance] is async.
+class RemoteConfig extends ChangeNotifier {
+  static const MethodChannel _channel =
+      const MethodChannel('plugins.flutter.io/firebase_remote_config');
+
+  static const String defaultValueForString = '';
+  static const int defaultValueForInt = 0;
+  static const double defaultValueForDouble = 0.0;
+  static const bool defaultValueForBool = false;
+
+  static const String fetchFailedThrottledKey = 'FETCH_FAILED_THROTTLED';
```

Comment:
I don't think the keys in this section should be part of the public API. Use `_`

Reply:
Done